jis-api/src/main/java/pl/edu/uj/JImageStream/model/Pixel.java

```diff
@@ -1,21 +1,33 @@
 package pl.edu.uj.JImageStream.model;
 
-public class Pixel {
+import java.awt.Point;
 
-    //todo add x y, integrate into predicate
+public class Pixel extends Point{
 
     private int red;
```

Comment:
could we hold colors already in int[] so getColors() method doesn't create new object every time it is called?